decentraland/unity-renderer
# fix: Hide UI should never hide the Start Menu

## What does this PR change?
Fix #1668 

- [x] Pressing any of the shortcuts keys that take you to the Start Menu should make it visible.
- [x] Development tag showing realm and network should also be hidden with this funcionality

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=fix/hide-ui-should-never-hide-the-start-menu
2. Press the [U] key.
3. Notice the UI in the world is hidden.
4. Press any of the Start Menu keys.
5. Notice the Start Menu is open correctly although the "Hidden UI" feature is activated.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
